green/loader.py

```diff
@@ -338,11 +342,12 @@ def toParallelTargets(suite, targets):
     modules = {x.module for x in proto_test_list}
     # Get the list of user-specified targets that are NOT modules
     non_module_targets = []
+    target: str
     for target in targets:
-        if not list(filter(None, [target in x for x in modules])):
+        if not list(filter(None, (target in x for x in modules))):
```

Comment:
Note this can probably be replaced with something like:
`if not any((target in x for x in modules)):`

But I need to fully grasp the original condition first.